migration-overview.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Migration Overview
+summary: This document describes how to migrate data from databases or data formats (CSV/SQL).
+category: reference
+---
+
+# Migration Overview
+
+This document describes how to migrate data to TiDB, including migrating data from MySQL and from CSV/SQL files.
+
+## Migrate from MySQL to TiDB
+
+To migrate data from MySQL to TiDB, it is recommended to use one of the following methods:
+
+- [Use Mydumper and TiDB Lightning](#use-mydumper-and-tidb-lightning-full-data) to migrate full data.
+- [Use TiDB Data Migration (DM)](#use-dm) to migrate full and incremental data.
+
+### Use Mydumper and TiDB Lightning (full data)
+
+#### Scenarios
+
+You can use Mydumper and TiDB Lightning to migrate full data when the data size is greater than 1 TB. If you need to replicate incremental data, it is recommended to [use DM](#use-dm) to create an incremental replication task.
+
+#### Migration method
+
+1. Use Mydumper to export the full MySQL data.
+2. Use TiDB Lightning to import the full data to TiDB. For details, refer to [Migrate from Mydumper Files](/migrate-from-mysql-mydumper-files.md).
+
+### Use DM
+
+#### Scenarios
+
+You can use DM to migrate full MySQL data and to replicate incremental data. Note that the size of the full data must be less than 1 TB. Otherwise, it is recommended to use Mydumper and TiDB Lightning to import the full data, and then use DM to replicate the incremental data.
```

Comment:
It is suggested that the full data less than 1TB, not must.

Reply:
updated in 73f64ae98590e8c63f441cdae81933d8cebab1bc